includes/AJAX.php

```diff
@@ -214,6 +214,11 @@ public function admin_complete_order() {
 	 * @since 2.0.0
 	 */
 	public function sync_products() {
+		// Inhibit on-demand full batch-api sync if the store has large product count.
```

Comment:
Let's tweak these comments, as this could be disabled based on various factors now that it's an opt-out hook.

E.g. `Allow opt-out of full batch-API sync, for example if store has a large number of products`.